addons/website/controllers/form.py

```diff
@@ -280,6 +280,25 @@ def insert_record(self, request, model, values, custom, meta=None):
                     message_type='comment',
                 )
 
+        if hasattr(record, '_message_log'):
+            _authenticate_message = False
+            if request.session.uid:
+                model_record = request.env[model_name]
+                user_email = request.env.user.email
+                form_email = values[model_record._mail_get_primary_email_field()]
+                if user_email != form_email:
+                    _authenticate_message = _("This %(model_name)s was submitted by %(user_name)s (%(user_email)s) on behalf of %(form_email)s",
+                        model_name=model.name, user_name=request.env.user.name, user_email=user_email, form_email=form_email)
+            else:
+                _authenticate_message = _("/!\\ EXTERNAL SUBMISSION - Customer not verified")
+
+            if _authenticate_message:
+                odoobot = request.env.ref('base.partner_root')
+                record._message_log(
+                    body=Markup('<div class="alert alert-info" role="alert">{message}</div>').format(message=_authenticate_message),
+                    author_id=odoobot.id,
+                )
```

Comment:
Since we just need the id, we could do that instead. 🙂 

```suggestion
                record._message_log(
                    body=Markup('<div class="alert alert-info" role="alert">{message}</div>').format(message=_authenticate_message),
                    author_id=request.env['ir.model.data']._xmlid_to_res_id('base.partner_root'),
                )
```